eoscostarica/guide.eoscostarica.io
# feat(frontend): add demux pattern page

### Add demux pattern page

### What does this PR do?

- resolves #444

### Steps to test
1. Go to "Learn EOSIO"
1. Open "Demux Pattern"
1. Check the content and metadata

#### CheckList
